pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -226,52 +247,46 @@ func (g *Gatherer) updateCache(ctx context.Context) {
 
 func (g *Gatherer) updateAlertsCache(ctx context.Context, metricsClient rest.Interface) error {
 	const logPrefix = "conditional gatherer: "
+	klog.Info(logPrefix + "updating alerts cache for conditional gatherer")
 
 	g.firingAlerts = make(map[string][]AlertLabels)
 
-	data, err := metricsClient.Get().AbsPath("federate").
+	data, err := metricsClient.Get().
+		AbsPath("api/v1/query").
```

Comment:
Is it better to use the common API versus the federate one?

Reply:
I just unified it with the integration tests so that we use the same API everywhere to eliminate strange bugs in the future